pkg/cmd/attestation/verification/tuf_test.go

```diff
@@ -0,0 +1,18 @@
+package verification
+
+import (
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestGitHubTUFOptions(t *testing.T) {
+	os.Setenv("CODESPACES", "true")
+	opts, err := GitHubTUFOptions()
+	assert.NoError(t, err)
+
+	assert.Equal(t, GitHubTUFMirror, opts.RepositoryBaseURL)
+	assert.NotNil(t, opts.Root)
+	assert.True(t, opts.DisableLocalCache)
```

Comment:
Consider switching from `assert` to `require` so test fails at first failed condition